### PR TITLE
[BBC-9059] Modal.mdx

### PR DESCRIPTION
**Pull request by florianeyzat**
## Description

Migrate Modal component

## How it was tested

Storybook + kairos project

### CODE REVIEW

File: src/modal/Modal.story.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Specifications
+
+WAI ARIA: https://www.w3.org/TR/wai-aria-practices-1.1/#dialog
+
+
+## Usage
+
+```jsx
+import { Modal } from '@blablacar/ui-library/build/Modal'
```
Comment (mcellier) [inline src/modal/Modal.story.mdx:50]: wrong path

Comment (mcellier) [reply src/modal/Modal.story.mdx:50]: - [ ] Fix import path

**Review by mcellier — COMMENTED** [review]
For this component, I'll like you to test in kairos ;)

File: src/modal/Modal.story.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      displayDimmer
+      isOpen={false}
+      closeButtonTitle="Close modal"
+      aria-labelledby="label1"
+      aria-describedby="description1"
+      size={ModalSize.MEDIUM}
+      isLoading={false}
+    />
+  </Story>
+  <Story name="fullscreen">
```
Comment (gabrielagabriel) [inline src/modal/Modal.story.mdx:26]: Could you add an Story by Preview?

File: src/modal/Modal.story.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+WAI ARIA: https://www.w3.org/TR/wai-aria-practices-1.1/#dialog
+
+
+## Usage
+
+```jsx
+import { Modal } from '@blablacar/ui-library/build/modal'
+
+<Modal />
```
Comment (gabrielagabriel) [inline src/modal/Modal.story.mdx:52]: Missing props here

**Review by mcellier — APPROVED** [review]

**Pull request merged**